cg/cli/workflow/microsalt/base.py

```diff
@@ -162,8 +164,14 @@ def run(
         ]
     microsalt_analysis_api.process.run_command(parameters=analyse_command, dry_run=dry_run)
 
-    if not sample_id and not dry_run:
-        microsalt_analysis_api.set_statusdb_action(case_id=case_id, action="running")
+    if sample_id or dry_run:
+        return
+
+    microsalt_analysis_api.set_statusdb_action(case_id=case_id, action="running")
+    try:
+        microsalt_analysis_api.submit_trailblazer_analysis(case_id=case_id)
+    except:
```

Comment:
Is it a process exception that is expected or what?

Reply:
If TB "service" is down, it can throw all kinds of errors.
OSError, google-oauth related errors, HTTPError
I have no way of knowing all kinds of edge cases actually. But this is general exception at microservice boundary